src/nodes/Image.tsx

```diff
@@ -211,54 +211,24 @@ export default class Image extends Node {
       view.dispatch(transaction);
     };
 
-  resizeImage = ({ node, getPos, width, height }) => {
-    const { view } = this.editor;
-    const { tr } = view.state;
-
-    const pos = getPos();
-    const transaction = tr.setNodeMarkup(pos, undefined, {
-      ...node.attrs,
-      width: Math.round(width),
-      height: Math.round(height),
-    });
-    view.dispatch(transaction);
```

Comment:
This call was to save the new width and height to the node image. Is deleting it intentional?

Reply:
nope, I thought it's unnecessary since after deleting that nothing is broken, gonna revert it back

Reply:
@hadielyakhni should be fixed now, let's recheck it